### DISCUSSION TITLE
No way of matching a specific port in rules

### DISCUSSION THREAD

**Discussion by mshunshin** [authelia/authelia#4927] [Q&A] [answered]
### Version

v4.37.5

### Deployment Method

Docker

### Reverse Proxy

Caddy

### Description

Within the configuration rules there appears to be no way of configuring a port (or at least no examples given in the documentation).

I have tried with domain: example.com:8080 but it doesn't match.

"No matching rule for subject username= groups= ip=100.113.14.60 and url https://example.com:8080/ (method GET)

### Reproduction

default_policy: deny
rules:
    - domain: example.com:8080
       policy: bypass

doesn't allow it to pass to to example.com:8080

### Expectations

the port can be specified as part of the rules (either as domain or elsewhere).

### Logs (Authelia)

```shell
Nil
```

### Pre-Submission Checklist

- [X] I agree to follow the [Code of Conduct](http://www.authelia.com/code-of-conduct)
- [X] This is a bug report and not a support request
- [X] I have read the security policy and this bug report is not a security issue or security related issue
- [X] I have either included the complete configuration file or I am sure it's unrelated to the configuration
- [X] I have provided all of the required information in full with the only alteration being reasonable sanitization in accordance with the [Troubleshooting Sanitization](https://www.authelia.com/r/sanitize) reference guide
- [X] I have checked for related proxy or application logs and included them if available
- [X] I have checked for related issues and checked the documentation

**Comment by james-d-elliott** [comment, marked as answer]
The domain is a match to the hostname of the URL, or the domain name portion. The port  is not relevant for this.
